src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java

```diff
@@ -0,0 +1,52 @@
+// to stop adding it:
+// git reset src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java
+package com.spartronics4915.frc2020.commands;
+
+import com.spartronics4915.frc2020.subsystems.PanelRotator;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Spin4TimesCommand extends CommandBase {
+
+    private PanelRotator mPanelRotator;
+    public double spins;
+    public String firstColor;
+    public String lastColor;
+    public String currentColor;
+    // You should only use one subsystem per command. If multiple are needed, use a CommandGroup.
+    public Spin4TimesCommand(PanelRotator mSubsystem) {
+      mPanelRotator = mSubsystem;
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+      spins = 0;
+      firstColor = mPanelRotator.getActualColor();
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+      currentColor = mPanelRotator.getActualColor();
```

Comment:
This looks solid, but will depend on how accurate the color sensor is looking at the spinning wheel. We'll also need to guarantee that the motor will stop the wheel in time.